src/styles/components/header/header.module.scss

```diff
@@ -0,0 +1,81 @@
+@import url('https://fonts.googleapis.com/css?family=Jost:wght@200..900');
+@import url('https://fonts.googleapis.com/css?family=Be+Vietnam:wght@200..900');
+
+.header {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--maroon, #5B1110);
```

Comment:
for all the color-related properties, take a look at the globals.scss file and use those variables instead of using absolute hex values because it facilitates any kind of project-level theme changes if there ever is one. Here's how this property would look if you used the var:

`background-color: var(--color-maroon);`